python/common/business-registry-dissolution/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "business-registry-dissolution"
+version = "0.1.0"
+description = ""
+authors = ["BrandonSharratt <brandon@daxiom.com>"]
+readme = "README.md"
+packages = [{include = "dissolution_service", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.13"
+flask = "^3.1.0"
+sqlalchemy = "^2.0.39"
+datedelta = "^1.4"
+business-model = {git = "https://github.com/bcgov/lear.git", subdirectory = "python/common/business-registry-model"}
+business-registry-account = {git = "https://github.com/bcgov/lear.git", subdirectory = "python/common/business-registry-account"}
+business-registry-common = {git = "https://github.com/BrandonSharratt/lear.git", subdirectory = "python/common/business-registry-common"}
```

Comment:
```suggestion
business-registry-common = {git = "https://github.com/lear/lear.git", subdirectory = "python/common/business-registry-common"}
```

Reply:
Sorry. It was suppose to be bcgov

Reply:
Thanks for catching that I had to use my version for the core Filing stuff but it'll be fine after this is merged